auth_token/backends.py

```diff
@@ -0,0 +1,21 @@
+from chamber.multidomains.auth.backends import ModelBackend
+
+from django.contrib.auth import get_user_model
+from django.utils import timezone
+
+from auth_token.models import DeviceKey
+
+
+class DeviceBackend(ModelBackend):
+    """
+    Authenticates device with UUID and login_token (password)
+    """
+    def authenticate(self, device_uuid=None, login_token=None, **kwargs):
+        if device_uuid and login_token:
+            device_key = DeviceKey.objects.filter(uuid=device_uuid).first()
+            if device_key and device_key.check_password(login_token):
+                user = get_user_model().objects.filter(id=device_key.user_id).first()
```

Comment:
ModelBackend má get_user to by ti asi mohlo stačit ne?

Reply:
použij jí

Reply:
good idea, fixed